### PR TITLE
Windowmasker update

### PR DESCRIPTION
**Pull request by DLBPointon**
I noticed that i didn't properly fix the bioconda version in windowmasker modules. It's also been requested I change the windowmasker mkcounts directory to match the yml.

I'm guessing that mkcounts isn't being tested as it is a "new" module (renamed).

- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [X] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [X] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/windowmasker/mkcounts/main.nf [hunk 1]
```diff
@@ -0,0 +1,55 @@
+process WINDOWMASKER_MKCOUNTS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::blast=2.14.0=h7d5a4b4_1"
```
Comment (nvnieuwk) [inline modules/nf-core/windowmasker/mkcounts/main.nf:5]:
```suggestion
    conda "bioconda::blast=2.14.0"
```

This is not needed in the conda declaration

File: modules/nf-core/windowmasker/ustat/main.nf [hunk 2]
```diff
@@ -2,7 +2,7 @@ process WINDOWMASKER_USTAT {
     tag "$meta.id"
     label 'process_low'
 
-    conda "bioconda::blast=2.14.0"
+    conda "bioconda::blast=2.14.0=h7d5a4b4_1"
```
Comment (nvnieuwk) [inline modules/nf-core/windowmasker/ustat/main.nf:5]:
```suggestion
    conda "bioconda::blast=2.14.0"
```

Same here :)

**Review by nvnieuwk — COMMENTED** [review]
Hi here are some comments, you also have to update the `tests/config/pytest_modules.yml` if you want the tests to trigger after renaming :)

**Review by muffato — COMMENTED** [review]
This commit should also delete all the `mk_counts` folders – and the GitHub diff should be detecting those as renames

**Comment by DLBPointon** [comment]
Old files now deleted, md5sum for versions.yml has also been removed. Thanks @muffato

**Pull request merged**